app/controllers/concerns/blacklight/catalog.rb

```diff
@@ -129,21 +119,22 @@ def store_preferred_view
     ##
     # Render additional response formats for the index action, as provided by the
     # blacklight configuration
-    #
-    # example:
-    #
-    #   config.index.respond_to.txt = Proc.new { render text: "A list of docs." }
-    #
-    # Make sure your format has a well known mime-type or is registered in
-    # config/initializers/mime_types.rb
+    # @param [Hash] format
+    # @note Make sure your format has a well known mime-type or is registered in config/initializers/mime_types.rb
+    # @example
+    #   config.index.respond_to.txt = Proc.new { render plain: "A list of docs." }
     def additional_response_formats format
       blacklight_config.index.respond_to.each do |key, config|
         format.send key do
           case config
           when false
             raise ActionController::RoutingError, 'Not Found'
           when Hash
-            render config
+            if config[:nothing] # nothing param  deprecated in rails, but still in configs
```

Comment:
This seems acceptable:

```
config.index.respond_to.txt = Proc.new { head :ok }
```

Also acceptable might be `config.index.respond_to.txt = { body: "" }`, or similar variations.